### PR TITLE
Add warning about js-ipfs being discontinued / point to Helia in a few places

### PR DESCRIPTION
**Pull request by ElPaisano**
# Describe your changes
Addresses https://github.com/ipfs/ipfs-docs/issues/1465 and [Part 1 of issue 1459](https://github.com/ipfs/ipfs-docs/issues/1459#issuecomment-1409131076)

## Checklist before requesting a review
- [ ] Passing the beta version of the **Check Markdown links for modified files** check. Action results can be viewed [here](https://github.com/ipfs/ipfs-docs/actions/workflows/action.yml).

## Checklist before merging
- [ ] Passing all required checks (The beta **Check Markdown links for modified files** check is not required)

### CODE REVIEW

**Comment by ElPaisano** [comment]
@2color I added this callout in the following places:

Install IPFS for JavaScript (js-ipfs)
IPFS Implementations
Create a simple chat app
IPFS repository guide > Protocol implementations

Any other spots you think we ought to add this in?

File: docs/concepts/ipfs-implementations.md [hunk 1]
```diff
@@ -5,6 +5,13 @@ description: "There isn't just one canonical IPFS implementation. Learn about th
 
 # IPFS Implementations
 
+:::warning js-ipfs being discontinued
+Development of the [js-ipfs project](https://github.com/ipfs/js-ipfs) is being discontinued to focus on [Helia](https://github.com/ipfs/helia), a leaner, more modular, modern implementation of IPFS in JavaScript slated for release in 2023. To learn more about Helia and the current state of IPFS in JS, see the [blog post](https://blog.ipfs.tech/state-of-ipfs-in-js/).
```
Comment (2color) [inline docs/concepts/ipfs-implementations.md:9]: I would move this either to below the table or into the js-ipfs row so it's not the first thing you read on the page.

Also, this is the first time I've seen `slated` used this way. Looking it up reveals that it's a north American word which probably explains why I haven't seen it. What do you think about a more neutral verb, e.g. scheduled or planned?

**Review by 2color — CHANGES_REQUESTED** [review]
The new warning element itself is great.

I would also add it to the following pages:
- https://docs.ipfs.tech/reference/js/api/
- https://docs.ipfs.tech/concepts/nodes/#js-ipfs

**Review by 2color — APPROVED** [review]

**Comment by 2color** [comment]
Either here or in a separate PR, e.g. #1360, we should add Helia to the list of implementation

**Comment by ElPaisano** [comment]
> Either here or in a separate PR, e.g. #1360, we should add Helia to the list of implementation



> Either here or in a separate PR, e.g. #1360, we should add Helia to the list of implementation

👍 made a note to address it as part of https://github.com/ipfs/ipfs-docs/issues/1465, will create a quick PR

**Pull request merged**